test/common.py

```diff
@@ -216,394 +206,299 @@ def tmp_digits_dataset(prefix: str,
 
 _SCORE_WITH_FACTORS_COMMON = " --source-factors {source_factors}"
 
-_EVAL_PARAMS_COMMON = "--hypotheses {hypotheses} --references {references} --metrics {metrics} {quiet}"
 
-_EXTRACT_PARAMS = "--input {input} --names target_output_bias --list-all --output {output}"
+def check_core_features(train_params: str,
```

Comment:
mh the function name is maybe a little too generic. Maybe we could name this `test_train_translate` analogous to `run_train_translate`.

Reply:
you are right, will rename

Reply:
sounds good.